docs/avi/setup_avi_ctrl.md

```diff
@@ -22,6 +22,15 @@ This is the simplest scenario with minimum requirements where the Management net
 
 <img src="net-flat.png" width="800"><br>
 
+In the screenshots below, it is assumed you are using the `Seperate Management network and VIP network` configuration.  However, in the instructions we will point out any distinctions if you choose those use this network setup.  Here are the configuration parameters in one of our contributor's (guess who) homelab.
```

Comment:
This sentence sounds odd to me:
`if you choose those use this network setup`

Reply:
I added some improved wording.